db/migrate/20210411213624_create_app_integrations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAppIntegrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :app_integrations do |t|
+      t.string :app_bundle, null: false, index: true
+      t.string :platform, null: false, index: true
+      t.boolean :active, null: false
```

Comment:
**suggestion (non-blocking):** As suggested above, `default: true` may make sense here since you always explicitly set that in the controller's `create` action right now.